### PR TITLE
HELPS-1763 Add ui.loopOpenHandler to example loops

### PR DESCRIPTION
**Pull request by MaggieMaSS**
### Changes made:

- Updated examples and added `ui.loopOpenHandler` to them.
### Additional notes:
This is the result of loops after typing `/` in the search bar
Screen Shot 2022-04-15 at 2 18 27 PM

### CODE REVIEW

File: ldk/javascript/examples/form-validation-loop/whisperManager/validation/index.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ const ssnRegex = /^[0-9]{4}$/;
 export const validateFormComponent = (
   component: Component,
   componentIds: ComponentIds,
-  value: string | boolean | number,
+  value: string | boolean | number | string[],
```
Comment (MaggieMaSS) [inline ldk/javascript/examples/form-validation-loop/whisperManager/validation/index.ts:24]: This change is not related to `loopOpenHandler`. But there was an error in this loop and adding` string[] `to `validateFormComponent.value` solved it

**Review by iangallOlive — APPROVED** [review]

**Review by racary-olive — APPROVED** [review]
